README.md

```diff
@@ -24,16 +15,16 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
 - `bin/rspec` - Run tests.
 - [Mailcatcher](http://mailcatcher.me/) is a good tool for catching emails in development.
 
-## Provision New VM
-- Provisioning relies upon SSH agent forwarding, so make sure you've set up an SSH
-  key for cubing@worldcubeassociation.org in order to rsync secrets.
-- `time ssh -A user@example.com 'sudo wget https://raw.githubusercontent.com/thewca/worldcubeassociation.org/master/scripts/wca-bootstrap.sh -O /tmp/wca-bootstrap.sh && sudo -E bash /tmp/wca-bootstrap.sh <environment>' - Where `environment` is one of `staging` or `production``
-
-## Deploy
+## Run in Vagrant (gets everything working, but is very slow, recommended only if you need to run the PHP portions of the website)
+- Install [Vagrant](https://www.vagrantup.com/), which requires
+  [VirtualBox](https://www.virtualbox.org/).
+- `vagrant up all` - Once the VM finishes initializing (which can take some time),
+  the website will be accessible at [http://localhost:2331](http://localhost:2331).
+  - Note: There are some minor [issues with development on Windows](https://github.com/thewca/worldcubeassociation.org/issues/393).
+- All emails will be accessible at `http://localhost:2332`.
+- Please take a look at this [wiki page](https://github.com/thewca/worldcubeassociation.org/wiki/Misc.-important-commands-to-know) for more detailed informations about the application's internals.
 
-See [this wiki](https://github.com/thewca/worldcubeassociation.org/wiki/Merging-and-deploying).
+# Production
 
-## Secrets
```

Comment:
I moved this over to our wiki: https://github.com/thewca/worldcubeassociation.org/wiki/Misc.-important-commands-to-know.